packages/sync/src/Functions.php

```diff
@@ -1,43 +1,65 @@
 <?php
+/**
+ * Utility functions to generate data synced to wpcom
+ *
+ * @package automattic/jetpack-sync
+ */
 
 namespace Automattic\Jetpack\Sync;
 
 use Automattic\Jetpack\Constants;
-/*
+
+/**
  * Utility functions to generate data synced to wpcom
  */
-
 class Functions {
 	const HTTPS_CHECK_OPTION_PREFIX = 'jetpack_sync_https_history_';
 	const HTTPS_CHECK_HISTORY       = 5;
 
+	/**
+	 * Return array of Jetpack modules.
```

Comment:
Would be great if each method could also have their access documented (`@access public`) and to have `@static` in case they're static.

Reply:
I tend not to add `@access` since WordPress uses that for "private" core functions

> @access: Only used for core-only functions or classes implementing “private” core APIs. If the element is private it will be output with a message stating its intention for internal use.

https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#1-functions-class-methods